<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigDeeperDialog</class>
 <widget class="QDialog" name="DigDeeperDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dig Deeper</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Process the Following File(s)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="selectedFileRadioButton">
            <property name="text">
             <string>Selected</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="checkedFileRadioButton">
            <property name="text">
             <string>Checked (#)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="listedFileRadioButton">
            <property name="text">
             <string>All (#)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>For the Following Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="thumbnailcheckBox">
          <property name="text">
           <string>Generate Thumbnails for All Image Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="videocheckBox">
          <property name="text">
           <string>Generate Thumbnails for All Video Files</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="hashcheckbox">
            <property name="text">
             <string>Calcuate Hash:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="md5radiobutton">
            <property name="text">
             <string>MD5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sha1radiobutton">
            <property name="text">
             <string>SHA1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sha256radiobutton">
            <property name="text">
             <string>SHA256</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="blake3radiobutton">
            <property name="text">
             <string>BLAKE3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="expandarchivescheckbox">
            <property name="text">
             <string>Expand Archives (zip)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="processButton">
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DigDeeperDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="exportButtonGroup"/>
 </buttongroups>
</ui>
